sway/meson.build

```diff
@@ -52,9 +52,10 @@ sway_sources = files(
 	'commands/smart_corner_radius.c',
 	'commands/create_output.c',
 	'commands/default_border.c',
+	'commands/default_dim_inactive.c',
 	'commands/default_floating_border.c',
 	'commands/default_orientation.c',
-	'commands/dim_inactive.c',
+    'commands/dim_inactive.c',
```

Comment:
Nit: match spacing

Reply:
My editor and git have different ideas of how may tabs to put in that file :)
Fixed in the latest commit.